server/chat/backend/agent/orchestrator/sub_agent.py

```
@@ -0,0 +1,504 @@
+            _update_db_terminal, agent_id, incident_id, user_id, "timeout",
+            tool_call_history=history,
+        )
+        return FindingRef(
+            agent_id=agent_id, role_name=role_name,
+            storage_uri=f"rca/{incident_id}/findings/{agent_id}.md",
+            status="timeout",
+            summary=f"Sub-agent {agent_id} ({role_name}) timed out after {timeout}s",
+        )
+
+
+async def _run(input_dict: dict) -> FindingRef:
```

Ruff (0.15.12)

[warning] Too many branches (15 > 12)

(PLR0912)